src/features/events/components/newsletter/index.js

```diff
@@ -68,35 +68,29 @@ const SubmitButton = styled(Button)`
 const url =
   '//prideinlondon.us6.list-manage.com/subscribe?u=8289d9ca2253b74574f849c73&id=a2423c3382&MERGE0='
 
-class NewsletterForm extends Component {
-  state = { value: '' }
+const NewsletterForm = () => {
+  const [emailValue, setEmailValue] = useState('')
 
-  handleChange = value => this.setState({ value })
+  handleChange = value => setEmailValue(value)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['handleChange' is not defined.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4755583)